Cards/MadGraph5_aMCatNLO/DY/DYto2L-2Jets_MLL-10to50_amcatnloFXFX-pythia8/DYto2L-2Jets_MLL-10to50_amcatnloFXFX-pythia8_cuts.f

```diff
@@ -1,4 +1,4 @@
-c *This file contains cuts(10~50) on the mll: line142-148*
+c ADDED: veto on invariant masses above 50 GeV in line 142-148
```

Comment:
Should we ask Sanghyun or someone if this is mg29x useable cuts.f file? I think at some point it changed but not sure in which version

Reply:
When I made the file, I checked out 299 and diffed cuts.f. The files are identical except the added lines.